mautic/mautic
# DPMMA-1157 Restrict password policy

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ N ]
| New feature/enhancement? (use the a.x branch)      | [ Y ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ Y ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:
This PR is a proposal to make more restrict password policy in Mautic.

Current password requirements: 
* Must contain at least 6 characters.

Password requirements after changes: 
* Password must be minimum eight characters,
* at least one uppercase letter, 
* at least one lowercase letter, 
* at least one number,
* at least one special character.


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try to add a new user with password that doesn't meet the requirements (should be rejected)
3. Add new user with password that meet the requirements
